### PR TITLE
Lxc path pre

### PR DESCRIPTION
**Pull request by kiorky**
let me look 2 minutes if it is ok

### CODE REVIEW

**Comment by kiorky** [comment]
tied to #23982

**Comment by kiorky** [comment]
ok !

File: salt/runners/lxc.py [hunk 1]
```diff
@@ -31,6 +31,16 @@
 }
 
 
+def _sanitized_kw(kwargs):
```
Comment (basepi) [inline salt/runners/lxc.py]: Please use `salt.utils.clean_kwargs()` instead

Comment (kiorky) [inline salt/runners/lxc.py]: i ll look if applicable, but yes sure

File: salt/states/lxc.py [hunk 2]
```diff
@@ -290,13 +299,19 @@ def present(name,
...
     Ensure a container is not present, destroying it if present
 
     name
         Name of the container to destroy
 
+    stop
+        stop before destroying
+        default: false
+
+        .. versionadded:: 2015.5.0
```
Comment (basepi) [inline salt/states/lxc.py]: should be 2015.5.2 (2015.5.1 was tagged internally today, so this kwarg won't be in a release until 2015.5.2)

**Pull request merged**